OsmAnd/src/net/osmand/plus/wikivoyage/data/TravelArticle.java

```diff
@@ -127,6 +127,10 @@ public String getRouteId() {
 		return routeId;
 	}
 
+	public String getGpxFileName() {
+		return !Algorithms.isEmpty(title) ? title : routeId;
+	}
+
```

Comment:
We also have [app.getTravelHelper().getGPXName(article)](https://github.com/osmandapp/Osmand/blob/0cabf8d2cafd71eb42fd012e7e3c4777f19008b2/OsmAnd/src/net/osmand/plus/wikivoyage/data/TravelObfHelper.java#L1039-L1042) will it lead to inconsistency or conflicts?

Reply:
fe3d143cbd